app/uk/gov/hmrc/gform/sharedmodel/formtemplate/SummarySection.scala

```diff
@@ -25,7 +25,8 @@ case class SummarySection(
   header: SmartString,
   footer: SmartString,
   continueLabel: Option[SmartString],
-  fields: Option[NonEmptyList[FormComponent]]
+  fields: Option[NonEmptyList[FormComponent]],
+  displayWidth: DisplayWidth.DisplayWidth = DisplayWidth.M
```

Comment:
Ok, can we make this other type? This doesn't represent same information as `DisplayWidth.DisplayWidth`. Also we should accept only value which have actually some effect. We are now silently doing nothing if user say for example `xs` etc.

Reply:
ok, makes sense. I will create a new type

Reply:
I introduced `SummaryDisplayWidth` type in both `gform` and `gform-frontend`